sql/2020/20_Caching/resource_age_party_and_type_wise_groups.sql

```diff
@@ -0,0 +1,45 @@
+#standardSQL
+# Age of resources party, type wise in groups.
+CREATE TEMPORARY FUNCTION toTimestamp(date_string STRING)
+RETURNS INT64 LANGUAGE js AS '''
+  try {
+    var timestamp = Math.round(new Date(date_string).getTime() / 1000);
+    return isNaN(timestamp) || timestamp < 0 ? null : timestamp;
+  } catch (e) {
+    return null;
+  }
+''';
+
+SELECT
+  client,
+  party,
+  resource_type,
+  COUNTIF(age_weeks IS NOT NULL) AS requests_total,

```

Comment:
It's not request `requests_total` so should this be:

```suggestion
  COUNT(0) as requests,
  COUNTIF(age_weeks IS NOT NULL) AS requests_with_last_modified,
```